Update to support 0.18

mrudat opened the pull request:
* Updated for 0.18
  * Renamed from "Fission and Fusion" to "Fission-and-Fusion", and wrote a helper to migrate the data from the old to the new name.
  * Added glow from base nuclear reactor to the inertial confinement reactor
  * Added name for fuel categories as these are now displayed.
  * Rewrote control.lua to hopefully perform better.
  * Allowed certain recipes to use productivity.
  * Allow fusion reactor interface update to be disabled.
  * Flagged RTG recycling and pressure cylinder unload recipes to not allow decomposition
  * Removed quickbar flags
  * Changed science pack names
  * Reformatted changelog

mrudat committed:
Update to support 0.17

  * Removed quickbar flags
  * Changed science pack names
  * Reformatted changelog

xGnoSiSx commented:
> 
> 
>     * Removed quickbar flags
> 
>     * Changed science pack names
> 
>     * Reformatted changelog

Hi,

it doesn't look like he's planning on a release or that he follows this any longer. Can you make a separate mod upload?

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 103.18kb -> 97.58kb (5.42%)

/graphics/reactor-interface-led.png -- 1.22kb -> 0.48kb (60.32%)
/graphics/icons/gas-deuterium.png -- 5.48kb -> 3.06kb (44.07%)
/graphics/fission-reactor-equipment-hr.png -- 96.48kb -> 94.04kb (2.54%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

mrudat committed:
Version 0.4.4

Changes:
  - Updated for 0.18
  - Added glow from base nuclear reactor to the inertial confinement reactor
  - Added name for fuel categories as these are now displayed.
  - Rewrote control.lua to hopefully perform better.
  - Allowed certain recipes to use productivity.
  - Allow fusion reactor interface update to be disabled.
Bugfixes:
  - Flagged RTG recycling and pressure cylinder unload recipes to not allow decomposition

mrudat committed:
Renamed mod

Renamed from "Fission and Fusion" to "Fission-and-Fusion", and wrote a helper to migrate the data from the old to the new name.

mrudat committed:
Merge pull request #1 from mrudat/imgbot

[ImgBot] Optimize images

mrudat committed:
Version: 0.4.5

- Make fusion generators input-output, just like the vanilla steam generators.
- Add an explicit dependency from the fusion power technology on space science packs
- Deuterium gas now has a fuel value of 6.2392J (what you would get if you just set it on fire), and fusion generators are now highly efficient (the difference between fusing and burning Deuterium).
- Reduced collision box on RTG to match sprite.
- no longer requires control.lua to make the fusion generators work.
- improve the alignment of entity sprites for RTG and fusion reactor.
- Hide no power icon for RTG - thanks to baslr
- Updated old changelog entries

mrudat committed:
Version 0.4.6

Fix graphics paths to refer to the _new_ mod name, rather than the old one.

mrudat committed:
Version 0.5.0

Features:
  - Add support for non-vanilla portable fusion generators
Changes:
  - Updated for 1.0
  - Fiddled with changelog again.
Bugfixes:
  - Fix stuff reported by luacheck.
